docs/spec/v1beta3/providers.md

```diff
@@ -1197,13 +1197,20 @@ secure communication. The secret must be of type `kubernetes.io/tls` or `Opaque`
 
 #### Providers supporting client certificate authentication
 
-The following webhook-based providers support client certificate authentication:
+The following providers support client certificate authentication:
 
 | Provider Type        | Description                    |
 |---------------------|--------------------------------|
 | `alertmanager`      | Prometheus Alertmanager        |
+| `azuredevops`       | Azure DevOps                   |
+| `bitbucket`         | Bitbucket                      |
+| `bitbucketserver`   | BitBucket Server/Data Center   |
 | `discord`           | Discord webhooks               |
 | `forwarder`         | Generic forwarder              |
+| `gitea`             | Gitea                          |
+| `github`            | GitHub                         |
+| `githubdispatch`    | GitHub Dispatch                |
```

Comment:
Looks like these two are not receiving support for client cert auth in this PR

Reply:
You're right, I thought I had already handled it, but I actually didn't. I'll take care of it now.

Reply:
Fixed via [5b7d807](https://github.com/fluxcd/notification-controller/pull/1146/commits/5b7d8076b530a09d6f572ef868e12a0c15de2deb)